setup.cfg

```diff
@@ -126,3 +126,210 @@ exclude=
   theme/views.py,
   hs_tools_resource/default_icon.py,
   hs_access_control/models/__init__.py
+  
+  # TODO: all of the following have issues that should be addressed that predated this commit. Need to go throught one at a time and fix them
```

Comment:
It will take me a while to address the existing flake8 issues. I can do that as part of this PR but thought it best to set this as a separate task since Alva seems to want these modifications in develop sooner rather than later.

Adding all of these ignored files does of course open us up to risk that we won't catch new flake8 problems in these files, so checking all of these ignored and removing them should be a priority in the near future

Reply:
I don't understand why flake8 is failing for all of these files because of this PR.  It looks like the issue is preexisting and if so, create an issue and address it there.

Reply:
I think it is because we updated our version of Flake8 and they expanded capacity to detect issues (and also included more PEP recommendations that weren't included before). This is my assumption I haven't researched...

Will open a new issue

Reply:
ah, that seems reasonable.